vispy/visuals/colorbar.py

```diff
@@ -679,5 +679,5 @@ def size(self, size):
         elif height == 0.:
             raise ValueError("height must be non-zero, not %s" % (height, ))
 
-        self._halfdim = (width, height)
+        self._halfdim = (width / 2, height / 2)
```

Comment:
I also didn't put any punctuation at the end of my sentence, so I didn't write goodly enough :)